packages/api/lambdas/jobs.js

```diff
@@ -130,7 +130,17 @@ async function cleanup() {
 
   const limit = pLimit(2);
 
+<<<<<<< HEAD
+  await Promise.all(
+    executions.slice(0, 400).map(
+      (ex) => limit(
+        () => checkExecution(ex.arn, ex.execution, ex.timestamp, esClient)
+      )
+    )
+  );
+=======
   await Promise.all(executions.slice(0, 400).map((ex) => limit(() => checkExecution(ex.arn, ex.execution, ex.timestamp, esClient))));
+>>>>>>> master
```

Comment:
@yjpa7145 this pr is not completely uninteresting - we should fix this for example